res/res_pjsip_refer.c

```diff
@@ -1710,9 +1989,16 @@ static int refer_incoming_refer_request(struct ast_sip_session *session, struct
 		return 0;
 	}
 
-	/* Determine if this is an attended or blind transfer */
-	if ((replaces = pjsip_param_find(&target_uri->header_param, &str_replaces)) ||
-		(replaces = pjsip_param_find(&target_uri->other_param, &str_replaces))) {
+	replaces = pjsip_param_find(&target_uri->header_param, &str_replaces);
+	if (!replaces) {
+		replaces = pjsip_param_find(&target_uri->other_param, &str_replaces);
+	}
+
+	/* Determine if this is handled externally or an attended or blind transfer */
+	trans_hand = pbx_builtin_getvar_helper(session->channel, "PJSIP_TRANSFER_HANDLING");
```

Comment:
This transitioned to being a dialplan function and part of the session, why the use of a dialplan variable still?

Reply:
The intention was to have a per endpoint ("global") option as well (without making it a full blown configuration option). Is there a better way to achieve this or should we only have the dialplan function?

Reply:
I really really don't want to add more special dialplan variables to Asterisk if it can be avoided. For global options in pjsip.conf the recommendation is templates that are inherited.